### DISCUSSION TITLE
HardLink PermissionError, Copying instead

### DISCUSSION THREAD

**Discussion by Silvers12** [pymedusa/Medusa#9233] [Q&A]
Hi, 
I just noticed that my disk size inflate... I activate debug log and try manuel post process.
And i see that in log : 
Error: PermissionError(1, 'Operation not permitted'). Copying instead

So all my series are twice ...
I don't know why because it's Medusa who create destination folder so all are medusa:medusa 777.

Have you an idea ? Or a clue ?
Thx a lot ;)

**Comment by p0psicles** [comment]
Your hardlinking on the same disk?

**Reply by Silvers12** [reply]
Yes.

But i found why ... Medusa have the good permission on the destination but not on sources ... 
rtorrent use temps directory. After that it move file into my folder series with 0666 and medusa try to hardlink series to series TV but he can't... If i change permission on sources it work. But rtorrent create file alway in 0666...
I have make an inotifywait for chmod when a file is create. For now it work...
